test/functional/Java16andUp/testClinkerFfi.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  Copyright (c) 2021, 2021 IBM Corp. and others
+
+  This program and the accompanying materials are made available under
+  the terms of the Eclipse Public License 2.0 which accompanies this
+  distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+  or the Apache License, Version 2.0 which accompanies this distribution and
+  is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+  This Source Code may also be made available under the following
+  Secondary Licenses when the conditions for such availability set
+  forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+  General Public License, version 2 with the GNU Classpath
+  Exception [1] and GNU General Public License, version 2 with the
+  OpenJDK Assembly Exception [2].
+
+  [1] https://www.gnu.org/software/classpath/license.html
+  [2] http://openjdk.java.net/legal/assembly-exception.html
+
+  SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
+<suite name="Java16andUp suite" parallel="none" verbose="2">
+	<test name="Jep389Tests_testClinkerFfi">
+		<classes>
+			<class name="org.openj9.test.jep389.downcall.PrimitiveTypeTests"/>
+			<class name="org.openj9.test.jep389.downcall.MultiCallTests"/>
+			<class name="org.openj9.test.jep389.downcall.MultiThreadingTests1"/>
+			<class name="org.openj9.test.jep389.downcall.MultiThreadingTests2"/>
+			<class name="org.openj9.test.jep389.downcall.MultiThreadingTests3"/>
+			<class name="org.openj9.test.jep389.downcall.MultiThreadingTests4"/>
+			<class name="org.openj9.test.jep389.downcall.InvalidDownCallTests"/>
+		</classes>
+	</test>
```

Comment:
Can we add this change in https://github.com/eclipse-openj9/openj9/blob/be92fa66592ddaeee8ce444f0c7b11ac95a893a1/test/functional/Java16andUp/testng.xml?

Reply:
Nope, this is for the new feature (JEP389) which has nothing to with JEP397.